### PR TITLE
Re-enable VirtHandlerRESTErrorsBurst and VirtHandlerRESTErrorsHigh tests

### PR DESCRIPTION
**Pull request by machadovilaca**
### What this PR does
Before this PR:

Flaky VirtHandlerRESTErrorsBurst and VirtHandlerRESTErrorsHigh were disabled

After this PR:

VirtHandlerRESTErrorsBurst and VirtHandlerRESTErrorsHigh added to the testing pipeline again

Fixes #

jira-ticket: https://issues.redhat.com/browse/CNV-40856

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
none
```

### CODE REVIEW

**Comment by machadovilaca** [comment]
1/1 passed

/test pull-kubevirt-e2e-k8s-1.29-sig-monitoring

**Comment by machadovilaca** [comment]
2/2 passed

/test pull-kubevirt-e2e-k8s-1.29-sig-monitoring

**Comment by machadovilaca** [comment]
3/3 passed

**Comment by sradco** [comment]
Please update the pr description

**Comment by avlitman** [comment]
/lgtm 

fix the description (:

**Comment by xpivarc** [comment]
/test pull-kubevirt-check-tests-for-flakes

@machadovilaca use this lane if you can as it has much higher change of detecting flakes

**Comment by machadovilaca** [comment]
@sradco @avlitman description updated

**Comment by machadovilaca** [comment]
pull-kubevirt-check-tests-for-flakes :heavy_check_mark:

File: tests/monitoring/component_monitoring.go [hunk 1]
```diff
@@ -127,14 +122,7 @@ var _ = Describe("[Serial][sig-monitoring]Component Monitoring", Serial, decorat
...
 			scales.RestoreAllScales()
-
-			time.Sleep(10 * time.Second)
-			alerts := []string{
-				virtOperator.downAlert, virtOperator.noReadyAlert, virtOperator.lowCountAlert,
-				virtController.downAlert, virtController.noReadyAlert, virtController.lowCountAlert,
-				virtApi.downAlert, virtApi.noReadyAlert, virtApi.lowCountAlert,
-			}
-			libmonitoring.WaitUntilAlertDoesNotExist(virtClient, alerts...)
+			waitUntilComponentsAlertsAreGone(virtClient)
```
Comment (avlitman) [inline tests/monitoring/component_monitoring.go:125]: nit: Gone is a bit not clear maybe DoesNotExist?

Comment (machadovilaca) [reply tests/monitoring/component_monitoring.go]: yup, was a bit too colloquial, changed

**Comment by sradco** [comment]
@avlitman please review

**Comment by avlitman** [comment]
/lgtm

No additional comments looks very good!

**Pull request merged**